application/values.yaml

```diff
@@ -73,17 +74,17 @@ deployment:
 
   # Init containers which runs before the app container
   initContainers:
-#      init-contaner:
-#        image: busybox
-#        imagePullPolicy: IfNotPresent
-#        command: ['/bin/sh']
+    #  init-contaner:
```

Comment:
This change is odd, alignment is wrong compared to some other changes. Can you please align correctly?

Reply:
@aslafy-z 
Sure, done